### ISSUE TITLE
Special Cards

### ISSUE THREAD

**Issue by D4NNY1000000** [Lululuz/FutBud#1]
Does this work with special cards?  Tried a Harry Kane out but didnt seem to work, confirmed he was for sale at a price using another account at the same time.

**Comment by Lululuz** [comment]
Yes should work, enable debug and check if he is found. Bot only buys if less than 2 results (default settings) are found to prevent loss. Also make sure you got the special card in your list not the normal one (Rarity tab)

**Comment by D4NNY1000000** [comment]
Just tried again with Arango RM totw_bronze and didnt work.  Only 1 was listed for 6000 at the time.
0 Auctions found.
https://utas.s3.fut.ea.com/ut/game/fifa16/transfermarket?start=0&num=13&maxb=6000&maskedDefId=-1291624319&type=player

**Comment by Lululuz** [comment]
Yep you are right. It doesn't work with the ID that is fetched by the FUT DB when you add a player. Somehow it always sends a negative defid. 
Will have a look at it but could take some time since my transfermarket is still locked and I need some time until my friends plays the 15 games to unlock it for me :D
